### PR TITLE
Public funded gross for expenditure on research and development

### PR DESCRIPTION
**Pull request by bailed22**
transformation and scripts reviewed on 
1. Datasets.json
2. Editions file
3. versions

### CODE REVIEW

File: src/store/metadata/stub/content/editions/versions/public-funded-gross_2023-10.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                    {
+                        "name": "geography",
+                        "datatype": "string",
+                        "titles": "Geography Type",
+                        "description": "The categorisation of the geography code."
+                    },
+                    {
+                        "name": "observation",
+                        "datatype": "decimal",
+                        "titles": "Gross disposable household income per head",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/editions/versions/public-funded-gross_2023-10.json:49]: The observation refers to gross disposable income, but the dataset is for gross for expenditure on R&D

File: src/store/metadata/stub/content/editions/public-funded-gross_2023-10.json [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          {
+            "name": "geography",
+            "datatype": "string",
+            "titles": "Geography Type",
+            "description": "The categorisation of the geography code."
+          },
+          {
+            "name": "observation",
+            "datatype": "decimal",
+            "titles": "Gross disposable household income per head",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/editions/public-funded-gross_2023-10.json:88]: The observation refers to gross disposable income, but the dataset is for gross for expenditure on R&D

**Review by GregoryJPavier — COMMENTED** [review]
Just need to change some metadata for the observation column.

File: src/store/metadata/stub/content/editions/public-funded-gross_2023-10.json (hunk 2 above)
Comment (bailed22) [reply src/store/metadata/stub/content/editions/public-funded-gross_2023-10.json:88]: Noted

Comment (bailed22) [reply src/store/metadata/stub/content/editions/public-funded-gross_2023-10.json:88]: resolved

**Review by GregoryJPavier — APPROVED** [review]

**Pull request merged**